configs/dictionaries/newbrowse_percentages.yaml

```
@@ -1,4 +1,4 @@
 ---
-A: 5
-B: 5
-Z: 90
\ No newline at end of file
+A: 50
+B: 50
+Z: 100
```

Shouldn't they add up to 100?

```suggestion
Z: 0
```

---

🤦‍♂️